src/content/docs/agents/nodejs-agent/index.mdx

```diff
@@ -19,13 +19,26 @@ redirects:
     * Enable [distributed tracing](/docs/apm/distributed-tracing/getting-started/introduction-distributed-tracing) to get visibility into modern distributed architectures.
 
     To keep your agent up to date and ensure you have access to the latest features, see the [Node.js agent release notes](/docs/release-notes/agent-release-notes/nodejs-release-notes).
+  
+  <ButtonGroup>
+    <ButtonLink
+    role="button"
+    to="https://newrelic.com/signup"
+    variant="primary"
+    >
+    Sign up for free
+    </ButtonLink>
+  </ButtonGroup>
+  No credit card required. Already have an account? [Login](http://one.newrelic.com/).
```

Comment:
@x8a, for some reason this page generated this: Gatsby.js development 404 page
There's not a page or function yet at /docs/agents/nodejs-agent/index

I tried various tricks but I wasn't able to fix this quickly. 

By the way, I don't think we need text if we have a button, so I think you can remove that text following the button.